### ISSUE TITLE
apt from i386 debian rootfs doesn't work

### ISSUE THREAD

**Issue by hrubymar10** [tbodt/ish#525]
# TL:DR
`apt` command doesn't work, because of `Illegal instruction`.

# Long story
I extracted debian rootfs from 32-bit docker image of debian stretch slim 9.11 .
I chrooted to that rootfs and it works fine, except of apt, which throws an `Illegal instruction` error.

# `dmesg` output after apt command
`[    0.000000] 15 illegal instruction at 0xf7966f07: d9 74 24 04 ba 00 04 00`

# output of `ldd /usr/bin/apt` 
```
(0xf7fd8000)
    libapt-private.so.0.0 => /usr/lib/i386-linux-gnu/libapt-private.so.0.0 (0xf7f6b000)
    libapt-pkg.so.5.0 => /usr/lib/i386-linux-gnu/libapt-pkg.so.5.0 (0xf7da5000)
    libstdc++.so.6 => /usr/lib/i386-linux-gnu/libstdc++.so.6 (0xf7c2b000)
    libgcc_s.so.1 => /lib/i386-linux-gnu/libgcc_s.so.1 (0xf7c0d000)
    libc.so.6 => /lib/i386-linux-gnu/libc.so.6 (0xf7a55000)
    libdl.so.2 => /lib/i386-linux-gnu/libdl.so.2 (0xf7a50000)
    libresolv.so.2 => /lib/i386-linux-gnu/libresolv.so.2 (0xf7a38000)
    libpthread.so.0 => /lib/i386-linux-gnu/libpthread.so.0 (0xf7a1b000)
    libz.so.1 => /lib/i386-linux-gnu/libz.so.1 (0xf7a00000)
    libbz2.so.1.0 => /lib/i386-linux-gnu/libbz2.so.1.0 (0xf79ee000)
    liblzma.so.5 => /lib/i386-linux-gnu/liblzma.so.5 (0xf79c2000)
    liblz4.so.1 => /usr/lib/i386-linux-gnu/liblz4.so.1 (0xf79ad000)
    libm.so.6 => /lib/i386-linux-gnu/libm.so.6 (0xf7958000)
    /lib/ld-linux.so.2 (0xf7fd9000)
```

# How to extract debian rootfs from docker image
1. Start Docker
2. Run `docker pull --platform=386 debian:9.11-slim`
3. Run `docker save debian:9.11-slim -o debian.tar.gz`
4. Extract `debian.tar.gz`
5. There is dir with long name and inside that directory is file called `layer.tar`
6. Airdrop `layer.tar` to your iSH
7. Extract it inside iSH using `tar xf layer.tar`
8. Enter chroot using `chroot /path/to/the/rootfs /bin/bash`

**Comment by tbodt** [comment]
Fixed in dca76175020111f9474843ad78601be26f30fa5a
